src/foam/nanos/export/GoogleSheetsOutputter.js

```diff
@@ -73,45 +54,81 @@ foam.CLASS({
     {
       name: 'outputObjectForProperties',
       type: 'StringArray',
+      code: async function(x, cls, obj, columnMetadata) {
+        var values = [];
+        var columnConfig = x.columnConfigToPropertyConverter;
+        for (var i = 0 ; i < columnMetadata.length ; i++ ) {
+          var val = columnConfig.returnValue(cls, columnMetadata[i].propName, obj);
+          values.push(await this.returnValueForMethadata(val, columnMetadata[i]));
+        }
+        return values;
+      }
+    },
+    {
+      name: 'returnMetadataForProperty',
+      code: function(prop, propName) {
+          var cellType = 'STRING';
+          var pattern = '';
+          if ( prop.cls_.id === "foam.core.UnitValue" ) {
```

Comment:
You can do: foam.core.UnitValue.isInstance(prop), save for below.

Reply:
fixed